test/lexical/collections/_LexicalFullyFeatured/index.ts

```diff
@@ -24,6 +25,19 @@ export const LexicalFullyFeatured: CollectionConfig = {
         features: ({ defaultFeatures }) => [
           ...defaultFeatures,
           TreeViewFeature(),
+          LinkFeature({
+            fields: ({ defaultFields }) => {
+              const modifiedFields = defaultFields.map((field) => {
+                if (field.name === 'newTab') {
+                  return { ...field, defaultValue: true } as CheckboxField
+                }
+
+                return field
+              })
+
+              return [...modifiedFields, { type: 'text', name: 'someText' }]
+            },
+          }),
```

Comment:
If the feature requires properties to be tested, it should have its own collection, [for the reason I explained here](https://github.com/payloadcms/payload/blob/735d6998041f88aeb53e2fac6b4139a76310810c/test/lexical/components/CollectionsExplained.tsx#L15). I made it `_LexicalFullyFeatured` because the tests were being run on a giant, non-isolated collection that was very difficult to modify because any change would break other tests.
Would you agree to create a LexicalLinkFeature collection?